src/helpers/chartHelpers.tsx

```diff
@@ -47,20 +56,11 @@ export function createLineChartDefs(layoutContext: LayoutContext, gradientKey: s
 
 
     const defs = seriesLineColors.map((lineColor, i) => {
-        if (nDomainMin !== undefined && nDomainMax !== undefined) {
             return (
                 <linearGradient id={`${gradientKey}${i}`} key={`${gradientKey}${i}`} x1="0%" y1="100%" x2="0%" y2="0%">
-                    {createStopsFromThresholds(lineColor, nDomainMin, nDomainMax, multiSeriesLineChartOptions?.thresholds)}
+                    {createStopsFromThresholds(lineColor, multiSeriesLineChartOptions?.thresholds, i)}
                 </linearGradient>
-            )
-        }
-
-        return (
-            <linearGradient id={`${gradientKey}${i}`} key={`${gradientKey}${i}`} x1="0" y1="0" x2="0" y2="100%">
-                <stop offset="0%" stopColor={lineColor} />
-                <stop offset="100%" stopColor={lineColor} />
-            </linearGradient>
-        )
```

Comment:
I simplified all of this down to a single return statement.  With the domain of the chart not being relevant to stops, it is correct in all cases to just call `createStopsFromThresholds`, which will create the 0% and 100% stops, and then do nothing if no thresholds are defined in between.